[2023-10-02T20:15:26.896+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: create_simple_table.create_dataset manual__2023-10-02T20:15:25.424404+00:00 [queued]>
[2023-10-02T20:15:26.908+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: create_simple_table.create_dataset manual__2023-10-02T20:15:25.424404+00:00 [queued]>
[2023-10-02T20:15:26.913+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-10-02T20:15:26.914+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-10-02T20:15:26.915+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-10-02T20:15:26.956+0000] {taskinstance.py:1300} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset> on 2023-10-02 20:15:25.424404+00:00
[2023-10-02T20:15:26.964+0000] {standard_task_runner.py:55} INFO - Started process 680 to run task
[2023-10-02T20:15:26.983+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'create_simple_table', 'create_dataset', 'manual__2023-10-02T20:15:25.424404+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/create_bq_table.py', '--cfg-path', '/tmp/tmpcqb29rvt']
[2023-10-02T20:15:26.992+0000] {standard_task_runner.py:83} INFO - Job 5: Subtask create_dataset
[2023-10-02T20:15:27.341+0000] {task_command.py:388} INFO - Running <TaskInstance: create_simple_table.create_dataset manual__2023-10-02T20:15:25.424404+00:00 [running]> on host 7db7086715ee
[2023-10-02T20:15:27.450+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=create_simple_table
AIRFLOW_CTX_TASK_ID=create_dataset
AIRFLOW_CTX_EXECUTION_DATE=2023-10-02T20:15:25.424404+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-02T20:15:25.424404+00:00
[2023-10-02T20:15:27.454+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-10-02T20:15:27.454+0000] {credentials_provider.py:323} INFO - Getting connection using `google.auth.default()` since no key file is defined for hook.
[2023-10-02T20:15:27.459+0000] {bigquery.py:451} INFO - datasetId was not specified in `dataset_reference`. Will use default value test_dataset.
[2023-10-02T20:15:27.460+0000] {bigquery.py:451} INFO - projectId was not specified in `dataset_reference`. Will use default value ***-tutorial-niko.
[2023-10-02T20:15:27.461+0000] {bigquery.py:461} INFO - Creating dataset: test_dataset in project: ***-tutorial-niko 
[2023-10-02T20:15:27.974+0000] {bigquery.py:1836} INFO - Dataset test_dataset already exists.
[2023-10-02T20:15:28.000+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=create_simple_table, task_id=create_dataset, execution_date=20231002T201525, start_date=20231002T201526, end_date=20231002T201528
[2023-10-02T20:15:28.046+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-10-02T20:15:28.061+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
